benchmarks/xla_flags_library.py

```diff
@@ -74,13 +74,6 @@
     " --2a886c8_chip_config_name=megachip_tccontrol"
 )
 
-ENABLE_SPARSECORE_OFFLOADING_BASE_FLAGS_PW = (
```

Comment:
Are we now providing these within the proxy flags - wl_config.model.xla_flags or pw_config.proxy_flags ?

Reply:
Yes, this was in particular removing the `2a886c8_chip_config_name` flag from the above config, so now that we have the "remove" functionality as part of `pathways_xla_flag_options` it's no longer necessary.